### PR TITLE
Update the code reloader design

### PR DESCRIPTION
**Pull request by rstacruz**
Yep! See: #1777.

image

image

image

I've added a bit of magic to parse `** (Xxxx) Yyyy` and display those up top—hope that's fine.

### CODE REVIEW

File: lib/phoenix/code_reloader.ex [hunk 1]
```diff
@@ -47,132 +56,220 @@ defmodule Phoenix.CodeReloader do
...
     </body>
     </html>
     """
   end
 
+  defp format_output(output) do
+    output
+    |> String.strip
+    |> Plug.HTML.html_escape
+    # |> String.replace(~r/(\n|^)(\*\* .*?)(\n|$)/, "\\1<strong>\\2</strong>\\3")
```
Comment (josevalim) [inline lib/phoenix/code_reloader.ex]: We can probably remove this line? :D

**Comment by josevalim** [comment]
😍

File: lib/phoenix/code_reloader.ex (hunk 1 above)
Comment (rstacruz) [inline lib/phoenix/code_reloader.ex]: aaand done

File: lib/phoenix/code_reloader.ex [hunk 2]
```diff
@@ -32,6 +32,15 @@ defmodule Phoenix.CodeReloader do
...
+    monospace_font: "menlo, consolas, monospace"
```
Comment (rstacruz) [inline lib/phoenix/code_reloader.ex]: I don't quite like how the logo base64 is repeated here (also seen in `Phoenix.Endpoint`). Any ideas?

Comment (josevalim) [inline lib/phoenix/code_reloader.ex]: I will think about it but nothing comes to mind.

**Comment by rstacruz** [comment]
I've no clue what to do about the test failures. :/

```
== Compilation error on file lib/phoenix/code_reloader/server.ex ==
** (CompileError) lib/phoenix/code_reloader/server.ex:69: unhandled operator ->
    (stdlib) lists.erl:1353: :lists.mapfoldl/3
    (stdlib) lists.erl:1354: :lists.mapfoldl/3
```

https://travis-ci.org/phoenixframework/phoenix/jobs/151743744

**Comment by josevalim** [comment]
@rstacruz you are using Elixir v1.2 I believe and that is Elixir v1.3 code. I will fix it in master.

**Comment by josevalim** [comment]
Fixed in master. If you rebase you should be good to go!

**Comment by rstacruz** [comment]
rebased!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by chrismccord** [comment]
Amazing work as usual @rstacruz! 😍😍😍

**Pull request merged**